source/routing/defining-your-routes.md

```diff
@@ -45,335 +45,48 @@ the case of `/`, the name `index`).
-
-You can reset the current "namespace" with the aptly named `resetNamespace: true` option.
-
-```app/router.js
-Router.map(function() {
-  this.route('post', { path: '/post/:post_id' }, function() {
-    this.route('edit');
-    this.route('comments', { resetNamespace: true }, function() {
-      this.route('new');
-    });
-  });
-});
-```
-
-This router creates five routes:
-
-<div style="overflow: auto">
-  <table>
-    <thead>
-    <tr>
-      <th>URL</th>
-      <th>Route Name</th>
-      <th>Controller</th>
-      <th>Route</th>
-      <th>Template</th>
-    </tr>
-    </thead>
-    <tr>
-      <td><code>/</code></td>
-      <td><code>index</code></td>
-      <td><code>app/controllers/index</code></td>
-      <td><code>app/routes/index</code></td>
-      <td><code>app/templates/index</code></td>
-    </tr>
-    <tr>
-      <td>N/A</td>
-      <td><code>post</code></td>
-      <td><code>app/controllers/post</code></td>
-      <td><code>app/routes/post</code></td>
-      <td><code>app/templates/post</code></td>
-    </tr>
-    <tr>
-      <td><code>/post/:post_id<sup>2</sup></code></td>
-      <td><code>post.index</code></td>
-      <td><code>app/controllers/post/index</code></td>
-      <td><code>app/routes/post/index</code></td>
-      <td><code>app/templates/post/index</code></td>
-    </tr>
-    <tr>
-      <td><code>/post/:post_id/edit</code></td>
-      <td><code>post.edit</code></td>
-      <td><code>app/controllers/post/edit</code></td>
-      <td><code>app/routes/post/edit</code></td>
-      <td><code>app/templates/post/edit</code></td>
-    </tr>
-    <tr>
-      <td>N/A</td>
-      <td><code>comments</code></td>
-      <td><code>app/controllers/comments</code></td>
-      <td><code>app/routes/comments</code></td>
-      <td><code>app/templates/comments</code></td>
-    </tr>
-    <tr>
-      <td><code>/post/:post_id/comments</code></td>
-      <td><code>comments.index</code></td>
-      <td><code>app/controllers/comments/index</code></td>
-      <td><code>app/routes/comments/index</code></td>
-      <td><code>app/templates/comments/index</code></td>
-    </tr>
-    <tr>
-      <td><code>/post/:post_id/comments/new</code></td>
-      <td><code>comments.new</code></td>
-      <td><code>app/controllers/comments/new</code></td>
-      <td><code>app/routes/comments/new</code></td>
-      <td><code>app/templates/comments/new</code></td>
-    </tr>
-  </table>
-</div>
-
-<small><sup>2</sup> `:post_id` is the post's id.  For a post with id = 1, the route will be:
-`/post/1`</small>
-
-The `comments` template will be rendered in the `post` outlet.
-All templates under `comments` (`comments/index` and `comments/new`) will be rendered in the `comments` outlet.
-
-The route, controller, and view class names for the comments resource are not prefixed with `Post`.
-
-### Multi-word Model Names
```

Comment:
I think it should be documented wherever we introduce the notion of the default behavior for a route like `this.route('post', { path: '/post/:post_id' });` loading the appropriate model. It has no context or meaning here.